src/__swaps__/screens/Swap/hooks/useSwapInputsController.ts

```diff
@@ -32,7 +32,7 @@ import { queryClient } from '@/react-query';
 import { divWorklet, equalWorklet, greaterThanWorklet, mulWorklet, toFixedWorklet } from '@/__swaps__/safe-math/SafeMath';
 
 function getInitialInputValues(initialSelectedInputAsset: ExtendedAnimatedAssetWithColors | null) {
-  const initialBalance = Number(initialSelectedInputAsset?.balance.amount) ?? 0;
+  const initialBalance = initialSelectedInputAsset ? Number(initialSelectedInputAsset?.balance.amount) : 0;
```

Comment:
you can use `||`, it checks any falsy value when `??` only checks `undefined` and `null`
```suggestion
  const initialBalance = Number(initialSelectedInputAsset?.balance.amount) || 0;
```